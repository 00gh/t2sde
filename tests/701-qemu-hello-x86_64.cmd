#!/bin/sh

[ "$QEMU" ] || exit 43

exe=usr/bin/qemu-x86_64

[ -e $SYSROOT/$exe ] || exit 42

$QEMU -L $SYSROOT $SYSROOT/$exe ${0%.cmd}.exe
