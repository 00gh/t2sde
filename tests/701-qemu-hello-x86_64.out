Hello x86_64
